source/installation/install-on-a-single-machine/install-on-a-single-machine.txt

```diff
@@ -41,6 +41,9 @@ If you are migrating from an Oracle database:
 
 - Download `21.6.0.0 of odjbc11.jar from the Oracle 21c <https://www.oracle.com/database/technologies/appdev/jdbc-drivers-archive.html>`_. 
 
+- (Optional) If you are migrating tables that contain the ``XML`` type, you must also download the 
+  `xdb.jar <https://www.oracle.com/database/technologies/appdev/jdbc-drivers-archive.html>`_ driver. 
```

Comment:
[suggestion, minimalism]

Maybe: "To migrate tables that contain the ``XML`` type..."

Reply:
Done.